### PR TITLE
Update modulemgr.h

### PR DESCRIPTION
**Pull request by Princess-of-Sleeping**

### CODE REVIEW

File: include/psp2/kernel/modulemgr.h [hunk 1]
```diff
@@ -110,21 +110,55 @@ int sceKernelStopModule(SceUID modid, SceSize args, void *argp, int flags, void
...
+// Maybe not present in 3.60
+/*
+ * int sceKernelSetSystemSwVersion(const SceKernelSystemSwVersion *version);
+ * SceUID __sceKernelCloseModule(SceUID uid, SceSize args, const void *argp, SceUInt32 flags);
+ * SceUID __sceKernelLoadModuleWithoutStart(const char *moduleFileName, SceUInt32 flags, const SceKernelLoadModuleOpt *pOpt);
+ * SceUID __sceKernelOpenModule(const char *moduleFileName, SceSize args, const void *argp, SceUInt32 flags);
+ * SceUID __sceKernelStartModule(SceUID uid, SceSize args, const void *argp, SceUInt32 flags);
+ * SceUID __sceKernelStopModule(SceUID uid, SceSize args, const void *argp, SceUInt32 flags);
+ * SceUID __sceKernelUnloadModuleWithoutStop(SceUID uid, SceUInt32 flags, const SceKernelUnloadModuleOpt *pOpt);
+ */
```
Comment (d3m3vilurr) [inline include/psp2/kernel/modulemgr.h:161]: do we need to add these comment?

File: include/psp2/kernel/modulemgr.h [hunk 2]
```diff
@@ -110,21 +110,55 @@ int sceKernelStopModule(SceUID modid, SceSize args, void *argp, int flags, void
...
- * @param[out] data - firmware information.
+ * @param[out] version - System sw version.
  *
  * @note - If you spoofed the firmware version it will return the spoofed firmware.
  */
-int sceKernelGetSystemSwVersion(SceKernelFwInfo *data);
+int sceKernelGetSystemSwVersion(SceKernelSystemSwVersion *version);
+
+// missing structs
+typedef struct SceKernelLoadModuleOpt SceKernelLoadModuleOpt;
```
Comment (d3m3vilurr) [inline include/psp2/kernel/modulemgr.h:133]: how about change the name to `SceKernelLoadModuleOption` like `SceKernelULMOption` and `SceKernelLMOption`

File: include/psp2/kernel/modulemgr.h [hunk 3]
```diff
@@ -110,21 +110,55 @@ int sceKernelStopModule(SceUID modid, SceSize args, void *argp, int flags, void
...
+typedef struct SceKernelUnloadModuleOpt SceKernelUnloadModuleOpt;
+typedef struct SceKernelLibraryInfo SceKernelLibraryInfo;
+
+SceUID _sceKernelLoadModule(const char *moduleFileName, SceUInt32 flags, const SceKernelLoadModuleOpt *pOpt);
```
Comment (d3m3vilurr) [inline include/psp2/kernel/modulemgr.h:137]:
```suggestion
SceUID _sceKernelLoadModule(const char *module_filename, SceUInt32 flags, const SceKernelLoadModuleOpt *option);
```
?

File: include/psp2/kernel/modulemgr.h (hunk 1 above)
Comment (Princess-of-Sleeping) [reply include/psp2/kernel/modulemgr.h:161]: If want to keep the 1.69 stuff for the low level hacker, else I don't think it's necessary

File: include/psp2/kernel/modulemgr.h (hunk 2 above)
Comment (Princess-of-Sleeping) [reply include/psp2/kernel/modulemgr.h:133]: I would choose `SceKernelLoadModuleOption`

File: include/psp2/kernel/modulemgr.h (hunk 3 above)
Comment (Princess-of-Sleeping) [reply include/psp2/kernel/modulemgr.h:137]: I forgot it. Just fix that

**Pull request merged**